internal/core/vds.go

```diff
@@ -108,6 +108,16 @@ type FenceMetadata struct {
 	Shape []int `json:"shape" swaggertype:"array,integer" example:"10,50"`
 } // @name FenceMetadata
 
+// @Description Attribute metadata
+type AttributeMetadata struct {
```

Comment:
Reminds me, maybe we could make a general struct with format and shape. I think these are used for all 3 endpoints? Mostly to centralise docs.

```
type Array struct {
    Format string ...
    Shape []int
}

type AttributeMetadata struct { 
    Array
}
```